package.json

```diff
@@ -50,7 +50,6 @@
     "check-dependency-version-consistency": "3.0.3",
     "dotenv-flow": "3.2.0",
     "husky": "8.0.3",
-    "just-install": "^1.0.11",
```

Comment:
I suppose people will need to install it manually now that it isn't installed with `yarn`

Reply:
Yes, this is sadly required 😞 (we considered this in the linked slack thread)